src/sonic-host-services/scripts/hostcfgd

```diff
@@ -1046,6 +1047,125 @@ class PamLimitsCfg(object):
                     "modify pam_limits config file failed with exception: {}"
                     .format(e))
 
+class DeviceMetaCfg(object):
+    """
+    DeviceMetaCfg Config Daemon
+    Handles changes in DEVICE_METADATA table.
+    1) Handle hostname change
+    """
+
+    def __init__(self):
```

Comment:
>def [](http://example.com/codeflow?start=4&length=3)

Suggest follow other Cfg class to define a `load` function, to get the boot up value of `self.hostname`. It may lead to one less update.